src/python/pants/backend/go/target_types.py

```diff
@@ -230,13 +230,27 @@ class GoBinaryDependenciesField(Dependencies):
     alias = "_dependencies"
 
 
+class GoPureField(BoolField):
+    alias = "pure"
```

Comment:
needs a default value

Reply:
Ack. Any thoughts on the rule graph issues?